src/common/conn/server.rs

```diff
@@ -171,7 +203,14 @@ where
         let cloned_exec = self.exec.clone();
         let http2_listener = HttpListener::new(self.http2_listener);
 
-        let srv = ViamServer::new(http2_listener, self.webrtc, cloned_exec);
+        let srv = ViamServer::new(
+            http2_listener,
```

Comment:
Yes you are right but the time is relatively short so i am inclined to leave it as is for now. In any case this should be done when we serving and on the condition http2 has been configured. We don't really need it for webrtc since we only have one interface open